src/pacmod3_nodelet.cpp

```diff
@@ -62,7 +64,9 @@ void Pacmod3Nl::onInit()
   vin_rpt_pub = nh_.advertise<pacmod3_msgs::VinRpt>("vin_rpt", 5);
 
   pub_tx_list.emplace(GLOBAL_RPT_CANID, std::move(global_rpt_pub));
```

Comment:
After std::move, these independent publish variables are no longer usable. How about create them as local variables in this function or use inplace constructing such as
```
pub_tx_list.emplace(GLOBAL_RPT_CANID, "vin_rpt", 5);
```

Reply:
No worries! if the work is done, all the better, just didn't want to hold the PR on something unrelated.
All good! merge at will